src/app/shared/modules/create-product-modal/create-product-modal.component.spec.ts

```diff
@@ -313,5 +337,33 @@ describe('CreateProductModalComponent', () => {
         ).toBe('test test');
       });
     });
+
+    describe('Create new Location Btn', () => {
```

Comment:
Only root describe suppose to start with Upper case, and all others depend on it. E.g.

```
describe('ExampleCalss', () => {
  describe('with some property', () => {
     it('does something', () => {
        ...
     });
  });

  describe('without some property', () => {
     it('does something', () => {
        ...
     });
  });
});
```

Turns to >>  2 cases:
1. ***ExampleCalss with some property does something***
2. ***ExampleCalss without some property does something***

The main goal is to make at the end a sentence that makes sense.